packages/open-payments/src/client/incoming-payment.ts

```diff
@@ -1,15 +1,21 @@
 import { HttpMethod, ResponseValidator } from 'openapi'
 import { BaseDeps, RouteDeps } from '.'
-import { IncomingPayment, getRSPath } from '../types'
-import { get } from './requests'
+import { IncomingPayment, getRSPath, CreateIncomingPaymentArgs } from '../types'
+import { get, post } from './requests'
 
 interface GetArgs {
   url: string
-  accessToken: string
+  accessToken?: string
+}
+
+interface PostArgs<T> {
+  url: string
+  body: T
 }
```

Comment:
I think both `GetArgs` and `PostArgs` should have the `accessToken` and have it be a required field